apps/web/src/views/tags.tsx

```diff
@@ -24,19 +24,33 @@ import Placeholder from "../components/placeholders";
 import { useSearch } from "../hooks/use-search";
 import { db } from "../common/db";
 import { ListLoader } from "../components/loaders/list-loader";
+import { useEffect } from "react";
+import { navigate } from "../navigation";
 
-function Tags() {
+type Props = {
+  location: "middle-pane" | "sidebar";
+};
+
+function Tags({ location }: Props) {
   useNavigate("tags", () => store.refresh());
   const tags = useStore((store) => store.tags);
   const refresh = useStore((store) => store.refresh);
   const filteredItems = useSearch("tags", (query) =>
     db.lookup.tags(query).sorted()
   );
 
+  useEffect(() => {
+    if (location === "sidebar") return;
+    tags?.item(0).then((item) => {
+      if (item && item?.item) {
+        navigate(`/tags/${item.item.id}`);
+      }
+    });
+  }, [tags, location]);
```

Comment:
I don't think this is needed.

Reply:
This is in case a user manually navigates to `app.notesnook.com/tags`, we can redirect them to the first tag. Plus this also prevents showing the tags listing in the middle pane.